nrfu_tests/test_hardware_fan_status.py

```diff
@@ -0,0 +1,146 @@
+# Copyright (c) 2022 Arista Networks, Inc.  All rights reserved.
+# Arista Networks, Inc. Confidential and Proprietary.
+
+"""
+Testcases for verification of fan status.
+"""
+
+import pytest
+from pyeapi.eapilib import EapiError
+from vane.logger import logger
+from vane.config import dut_objs, test_defs
+from vane import tests_tools
+
+TEST_SUITE = "nrfu_test"
+
+
+@pytest.mark.nrfu_test
```

Comment:
```suggestion
@pytest.mark.nrfu_test
@pytest.mark.system
```
need to add this marker as well